annotate.html

```diff
@@ -0,0 +1,370 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+   "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+<META NAME="viewport" CONTENT="width=device-width, initial-scale=1.0">
+    <title>Metamath $j annotations</title>
+  </head>
+  <body>
+<TABLE BORDER=0 WIDTH="100%"><TR>
+<!-- ...TD style="line-height:100%"... -->
+
+<TD NOWRAP ALIGN=LEFT VALIGN=MIDDLE WIDTH="25%"> <FONT SIZE=-2
+FACE=arial><A HREF="../index.html"><IMG SRC="mm.gif" BORDER=0
+ALT="Home" HEIGHT=32 WIDTH=32 ALIGN=LEFT><IMG SRC="spacer.gif" BORDER=0
+ALT="" HEIGHT=32 WIDTH=1 ALIGN=LEFT>Metamath<BR>Home</A></FONT></TD>
+
+<TD NOWRAP ALIGN=CENTER VALIGN=MIDDLE><FONT SIZE="+3"
+COLOR="#006633"><B>Metamath $j annotations</B></FONT></TD>
+
+<TD WIDTH="25%"> &nbsp; </TD>
+
+</TR></TABLE>
+
+<HR NOSHADE SIZE=1>
+<p>As noted in section 4.4.3, "Additional Information Comment ($j)",
+of the metamath specification, we have a way to include information in
+metamath proof files which are not needed for verifying, but which are
+useful for other tools like proof assistants, definition checkers,
+conversion to and from other proof formats, and the like. Although the
+metamath specification has some examples, it seems better to document
+the full list here because they aren't part of the specification in
+the sense of being needed for verification.</p>
+
+<h2>Syntax and definitions</h2>
+<code>$j syntax ’TYPECODE’ ...;</code>
+
+<p>State that TYPECODE is a typecode. That is, it is a constant symbol
+which appears in the left most position in statements. This form is
+specifically for variable typecodes, which means they have one or
+more $f statements and are usually omitted from HTML display. Syntax
+axioms (which have this typecode) should not have any repeated
+variables or $d or $e hypotheses.</p>
+
+<p>This command is repeatable by passing a list of typecodes.</p>
+
+<code>$j syntax ’SYNTAX’ for 'TYPECODE';</code>
```

Comment:
These are not the same quotes... Is that wanted ?

Reply:
Good catch, I might have mangled these copying from github (or something). If you look at the `$j` statements in *.mm they contain the https://www.fileformat.info/info/unicode/char/27/index.htm character so that is what we should use here.